testsuite/WIKI.md

```diff
@@ -0,0 +1,198 @@
+The unit tests and the test data are bundled together in the package **MDAnalyisTests-_release_**. In order to run the tests, this package must be installed in addition to MDAnalysis.
+
+The tests also rely on the `pytest` and `numpy` packages, and require both to run.
+
+# Users
+Install [MDAnalysisTests](MDAnalysisTests) via
+```
+pip install --upgrade MDAnalysisTests
+```
+or download the tar file, unpack, and run `python setup.py install`.
+
+Run the tests by invoking
+```
+python -c 'from MDAnalysis.tests import test; test()'
+```
+
+All tests should pass (i.e. no **FAIL**, **ERROR**); *SKIPPED* or *XFAIL* are ok. For anything that fails or gives an error [ask on the user mailing list](http://users.mdanalysis.org) or [raise an issue](/MDAnalysis/mdanalysis/issues).
+
+# Developers #
+
+All tests should pass (i.e. no **FAIL**, **ERROR**); *SKIPPED* or *XFAIL* are ok. For anything that fails or gives an error **fix your code** (or *raise an issue*).
+
+**Do not push code that fails to the development branch.** Instead, push it to a feature or issue branch. It will run automatically through the unit tests by travis-ci and it will be available for comment and discussion by other developers.
+
+## Recommended ##
+Use the tests from the [git source repository](Source), which are located in the [testsuite/MDAnalysisTests](https://github.com/MDAnalysis/mdanalysis/tree/develop/testsuite) directory:
+```
+cd testsuite/MDAnalysisTests
+pytest  --disable-pytest-warnings --numprocesses 4
+```
+(Try increasing the number of processes; with 24 processes on 12 cores (+hyperthreading) this took ~40 seconds; in serial it takes ~30 min).
+
+To run specific tests just specify the path to the test file:
+```
+pytest test_analysis.py
+```
+
+Specific test classes inside test files, and even specific test methods, can also be specified:
+```
+# Test the entire TestContactMatrix class
+pytest test_analysis.py::TestContactMatrix
+# Test only test_sparse in the TestContactMatrix class
+pytest test_analysis.py::TestContactMatrix::test_sparse
```

Comment:
I imagine so, and I'm planning to the next time I need to test. We should definitely add an example for `-k` too.

I'll be sticking to `--pyargs MDAnalysisTests`; don't you think it should be the recommended way rather than have the user find wherever MDAnalysisTests is installed? Granted this will likely be run by devs that should actually know what they're doing, but I like the idea of an independent testing environment where to keep logs and such.